README.md

```diff
@@ -70,15 +70,15 @@ npx hardhat verify --network bscmain {contract address} "parameter 1" "parameter
 ·······················|························|·············|·············|··············|···············|··············
 |  DixelClubV2Factory  ·  updateImplementation  ·          -  ·          -  ·       28949  ·            3  ·          -  │
 ·······················|························|·············|·············|··············|···············|··············
-|  DixelClubV2NFT      ·  addWhitelist          ·      56672  ·     218145  ·      108830  ·           29  ·          -  │
+|  DixelClubV2NFT      ·  addWhitelist          ·      39044  ·     189236  ·      167735  ·           29  ·          -  │
```

Comment:
Oh it is actually much cheaper than using `EnumerableMapping` I tried in the first place, but it seems it's still ~60% more expensive than a simple array implementation.

I'm thinking of a hacky solution @assafom  suggested here: #8 
What do you think about that one?

Reply:
The maximum minimum gas requirement has been reduced, and the average is high.

Reply:
I think we need to check the test case that manages more than 100 lists. The number of lists is too small to check the edge case at this time.

Reply:
True. I'll add some test cases for a big list first and let's see.

Reply:
Added some benchmark test:
https://github.com/Steemhunt/dixel-v2-contract/commit/6495dd6b59ce92cfce76a26d222bc1a3beccfb5a

Threads for whitelist optimization:
https://github.com/Steemhunt/dixel-v2-contract/issues/2#issuecomment-1146777315